src/algebra/big_operators/finprod.lean

```diff
@@ -408,6 +408,20 @@ begin
   simp [mul_support_mul]
 end
 
+/-- If the multiplicative supports of `f` and `g` are finite, then the product of `f i * (g i)⁻¹`
+equals the product of `f i` multiplied by the inverse of the product over `g i`. -/
+@[to_additive] lemma finprod_mul_inv_distrib {M' : Type*} [comm_group M'] {f g : α → M'}
+  (hf : (mul_support f).finite) (hg : (mul_support g).finite) :
+  ∏ᶠ i, (f i * (g i)⁻¹) = (∏ᶠ i, f i) * (∏ᶠ i, g i)⁻¹ :=
+by rw [finprod_mul_distrib hf ((mul_support_inv g).symm.rec hg), finprod_inv_distrib]
+
+/-- If the multiplicative supports of `f` and `g` are finite, then the product of `f i / (g i)`
+equals the product of `f i` divided by the product over `g i`. -/
+@[to_additive] lemma finprod_div_distrib {M' : Type*} [comm_group M'] {f g : α → M'}
+  (hf : (mul_support f).finite) (hg : (mul_support g).finite) :
+  ∏ᶠ i, f i / g i = (∏ᶠ i, f i) / ∏ᶠ i, g i :=
+by simp [div_eq_mul_inv, finprod_mul_inv_distrib hf hg]
```

Comment:
This lemma _is_ worth having, but I still think the one above probably isn't.

Can you also add `finprod_mem_div_distrib`?

Reply:
But whats the point of combining two proofs into one if they are already broken up? Its just removing a small lemma someone might want to rewrite with or search for with library_search, whats the problem you are trying to solve?

Reply:
It required some more additions but there is now a mem version